script/bootstrap.py

```diff
@@ -12,6 +12,7 @@
 SOURCE_ROOT = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
 VENDOR_DIR = os.path.join(SOURCE_ROOT, 'vendor')
 PYTHON_26_URL = 'https://chromium.googlesource.com/chromium/deps/python_26'
+CRACKPAD_URL = 'https://github.com/hokein/crashpad.git'
```

Comment:
Can you move this repo to atom organization? I'll add you as collaborator so you can still push to it.